components/entry/entry-form.tsx

```diff
@@ -81,7 +82,7 @@ const SortableItem = ({
   };
 
   return (
-    <div ref={setNodeRef} className={cn("bg-background flex gap-x-1 rounded-lg border items-center", type === "object" ? "px-2 py-4" : "px-1 py-2", isDragging ? "z-50" : "z-10")} style={style}>
+    <div ref={setNodeRef} className={cn("bg-background flex gap-x-1 rounded-lg border items-start", type === "object" ? "px-2 py-4" : "px-1 py-2", isDragging ? "z-50" : "z-10")} style={style}>
```

Comment:
This makes UI jump less when expanding.

Before change:

https://github.com/user-attachments/assets/6058aba9-5a88-4567-a4bb-f9fb7eaccd18


After change:


https://github.com/user-attachments/assets/0e2f7538-c604-4c40-9b1e-93143ff2a332